src/core/public/application/types.ts

```diff
@@ -194,7 +194,9 @@ export type AppUpdatableFields = Pick<AppBase, 'status' | 'navLinkStatus' | 'too
  * see {@link ApplicationSetup}
  * @public
  */
-export type AppUpdater = (app: AppBase) => Partial<AppUpdatableFields> | undefined;
+export type AppUpdater = (
+  app: AppBase
+) => Partial<AppUpdatableFields & { activeUrl: string }> | undefined;
```

Comment:
Had to add this to make sure the type was correct when using it with the KibanaUrlTracker